package.json

```diff
@@ -85,10 +85,10 @@
     "@angular/platform-browser-dynamic": "^11.2.14",
     "@angular/router": "^11.2.14",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "6.18.3-ConsoleLogs.1",
+    "@hmcts/ccd-case-ui-toolkit": "6.18.2-exui-659-mv-upgrade-rc1",
```

Comment:
version should be incremented either in minor version or patch element.

Reply:
Can you please explain what that means? Not sure which minor or patch. I got these changes from @RiteshHMCTS PR

Reply:
The ui toolkit version looks to be going from `6.18.3` down to `6.18.2`. In fact, the ui toolkit version should go up to `6.18.4`. The new MV version is containing some new features, and therefore this change will bring some new features to UI Toolkit itself. Therefore, ui toolkit's "minor version" should be incremented.